docs/conf.py

```diff
@@ -3,6 +3,11 @@
 import sys
 from os import path
 
+version = ''
+# Centralize environment variables
+JINA_VERSION = os.environ.get('JINA_VERSION')
+SPHINX_MULTIVERSION_VERSION = os.environ.get('SPHINX_MULTIVERSION_VERSION', version)
```

Comment:
this is not what is intended. See that `version` is used later, and thus this reading should be placed after the `version` var initialization

Reply:
Okay I let me remove that...